folio-security/src/test/resources/log4j2.properties

```diff
@@ -6,6 +6,10 @@ appender.console.type=Console
 appender.console.name=STDOUT
 appender.console.layout.type=PatternLayout
 appender.console.layout.pattern=%d{HH:mm:ss} %-5p %-20.20C{1} %m%n
-rootLogger.level=info
+
+# To reliably capture output from Log4j2 (org.springframework.boot.test.system.OutputCaptureExtension.class)
+appender.console.follow = true
+
+rootLogger.level=debug
```

Comment:
Do we want to have debug the default level?

Reply:
for tests - yes, to be able to validate debug log level logs